code/__DEFINES/layers.dm

```diff
@@ -88,6 +88,8 @@
 #define LIGHTING_PLANE 15
 #define LIGHTING_LAYER 15
 
+#define RAD_TEXT_LAYER 21
```

Comment:
Why define this to be the same as the hud layer instead of just putting it on the hud layer?